templates/helm/orchestration-workflows/encode/templates/encode.yaml

```diff
@@ -34,6 +35,14 @@ spec:
           - kubeSecretKey: AWS_SECRET_ACCESS_KEY
             path: {{ $path }}
             vaultKey: secret_access_key
+      {{- if $smokeTest }}
+      - secretName: smoke-test-secrets
+        nameSpace: encode
+        vals:
+          - kubeSecretKey: slack-url
+            path: secret/dsde/monster/{{ .Values.env }}/slack-notifier
+            vaultKey: monster-ci-url
```

Comment:
The URL I generated is routing messages to #monster-ci. Do we want to keep using that channel? Do we want to reuse one of our existing channels? Or create a brand-new thing?

Reply:
Using #monster-ci makes sense to me.